esx_service/vmci/vmci_client.c

```diff
@@ -262,38 +265,44 @@ vsock_get_reply(be_sock_id *s, be_request *r, be_answer* a)
    }
 
    // Now get the reply (blocking, wait on ESX-side execution):
-
    // MAGIC:
    b = 0;
    ret = recv(s->sock_id, &b, sizeof b, 0);
    if (ret == -1 || ret != sizeof b ) {
+      snprintf(a->errBuf, ERR_BUF_LEN, "Failed to receive magic data: received %d expected %d bytes\n",
```

Comment:
we still do not set errno for 0 <= ret < sizeof b (this applies to all places where we check the exected vs. received. 

Do you plan to put it in a separate PR ?